packages/core-sdk/resolvers/account-resolvers/interface/query.graphql

```diff
@@ -1,72 +1,130 @@
 #import { Token } into TokenResolver from "ens/interface.token-resolvers.defiwrapper.eth"
 
 type Query {
-  getTokenBalances(address: String!): [TokenBalance!]!
+  getTokenBalances(accountAddress: String!): TokenBalancesList!
 
-  # TODO
-  # getTokenTransfers(address: String!): TransfersList!
+  getTokenTransfers(
+    accountAddress: String!
+    tokenAddress: String!
+    vsCurrency: String!
```

Comment:
What is `vsCurrency`?

Reply:
It's the terminology used by coingecko to describe quote currency in which you want price